Codigo/Declares.bas

```diff
@@ -31,6 +31,8 @@ Option Explicit
 ''
 ' Modulo de declaraciones. Aca hay de todo.
 '
+Public Const API_URL_SERVER As String = "https://localhost:9090"
```

Comment:
if we only use this on APiConnection I would move this to APiconneciton.bas and keep it private, it's already hard to maintain Declares.bas, we should only need to add stuff that we need in multiple files